app/src/main/java/de/rochefort/childmonitor/DiscoverActivity.kt

```diff
@@ -0,0 +1,247 @@
+/*
+ * This file is part of Child Monitor.
+ *
+ * Child Monitor is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Child Monitor is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Child Monitor. If not, see <http://www.gnu.org/licenses/>.
+ */
+package de.rochefort.childmonitor
+
+import android.app.Activity
+import android.content.Intent
+import android.net.nsd.NsdManager
+import android.net.nsd.NsdManager.DiscoveryListener
+import android.net.nsd.NsdServiceInfo
+import android.net.wifi.WifiManager
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.AdapterView
+import android.widget.AdapterView.OnItemClickListener
+import android.widget.ArrayAdapter
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ListView
+import android.widget.Toast
+
+val TAG = "ChildMonitor"
```

Comment:
If we are going to use a global TAG it should somewhere else. If it is used in DiscoverActivity only we should move it to a companion object

Reply:
Will do